### PR TITLE
feat: Comments & Share Post

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
- Made single-page UI for post
- Share Post
- Add Comment
- Delete Comment
- Solved bug in follow-unfollow users

### CODE REVIEW

File: src/services/userService.ts [hunk 1]
```diff
@@ -70,22 +70,46 @@ export const decrementLike = createAsyncThunk("user/decrementLike", async({postI
...
     } catch(err) {
         console.log(err)
     }
 }))
 
 
-export const removeFollowing = (createAsyncThunk("user/addFollower", async({userId, item, authToken, authItem}:{userId:string, item:any, authToken:string, authItem:any}) => {
+export const removeFollowing = (createAsyncThunk("user/removeFollowing", async({userId, item, authToken, authItem}:{userId:string, item:any, authToken:string, authItem:any}) => {
     try {
+        console.log("removed")
```
Comment (aniketsingh13) [inline src/services/userService.ts]: remove console here

**Comment by ark2002** [comment]
image

while signing in with the credentials (guest ) i got this error

**Review by aniketsingh13 — APPROVED** [review]
code look good to merge

File: src/components/common/moreOptions/MoreOptionsModal.tsx [hunk 2]
```diff
@@ -1,17 +1,23 @@
 import { useAppDispatch } from "hooks";
 import React, { SetStateAction } from "react";
 import { deletePost } from "services/postsServices";
+import { deleteComment } from "services/userService";
+import { CommentDataType } from "types/user.types";
 
 export const MoreOptionsmOdal = ({
```
Comment (ark2002) [inline src/components/common/moreOptions/MoreOptionsModal.tsx:7]: could use a proper naming MoreOptionModal

File: src/pages/singlePost/index.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        </div>
+        <div className="xs:hidden md:block block lg:col-span-3 md:col-span-4">
+          <UserSidebar />
+        </div>
+  
+        {/* TODO: For future reference */}
+        {/* <div className="col-span-8 p-3 bg-green-100">
+          <Footer />
+        </div> */}
+
```
Comment (ark2002) [inline src/pages/singlePost/index.tsx:29]: could remove comments

**Review by ark2002 — COMMENTED** [review]
there was a glitch in ur bookmarks too ...
if there are multiple posts bookmakrs and u go to bookm ark and click on the unbookmark button the post disappears but also disables the bookmarked selected for the other posts 

.. was gonna send an ss for refrence but the login error didnt let me login again

**Comment by Vishruta-Patil** [comment]
> image
> 
> while signing in with the credentials (guest ) i got this error

This issue was due to the firebase quota. But now it is resolved
Thanks for the review

**Comment by Vishruta-Patil** [comment]
> there was a glitch in ur bookmarks too ... if there are multiple posts bookmakrs and u go to bookm ark and click on the unbookmark button the post disappears but also disables the bookmarked selected for the other posts
> 
> .. was gonna send an ss for refrence but the login error didnt let me login again

Thanks for pointing out! It is resolved now

**Pull request merged**